### PR TITLE
Reduce release churn

### PR DESCRIPTION
**Pull request by orta**
Trying to remove more post deployment diff faff.

#trivial

### CODE REVIEW

File: Example/Podfile.lock [hunk 1]
```diff
@@ -255,7 +255,7 @@ SPEC CHECKSUMS:
   Artsy-UIButtons: 3c396f0fad352a7b0332100e0ffcb0ca577e0082
   boost-for-react-native: 39c7adb57c4e60d6c5479dd8623128eb5b3f0f2c
   DoubleConversion: ebb6747c5b66026ad4f97b789c3ceac6f18e57a6
-  Emission: 8ffafeae9d93bdf019e60e8a21351f70d1d63ec7
+  Emission: f18165964b629d103bdba726db336903584eb2a3
```
Comment (alloy) [inline Example/Podfile.lock]: Err, so did my addition to the script not work?

**Comment by alloy** [comment]
Thanks!

File: Example/Podfile.lock (hunk 1 above)
Comment (orta) [reply Example/Podfile.lock]: Looks like it to me alas

Comment (alloy) [reply Example/Podfile.lock]: Ok, I’ll have to think a bit longer about this then.

Comment (ashfurrow) [reply Example/Podfile.lock]: Opened a ticket to address this: https://artsyproduct.atlassian.net/browse/LD-485

Comment (alloy) [reply Example/Podfile.lock]: Thanks!

**Pull request merged**